spec/rubocop/cop/rspec/contain_exactly_spec.rb

```diff
@@ -0,0 +1,26 @@
+RSpec.describe RuboCop::Cop::RSpec::ContainExactly do
+  subject(:cop) { described_class.new }
+
+  it 'flags `contain_exactly` with only splat arguments' do
+    expect_offense(<<-RUBY)
+      it { is_expected.to contain_exactly(*array1, *array2) }
+                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Prefer `match_array` when matching array values.
+    RUBY
+  end
+
+  it 'does not flag `match_array`' do
+    expect_no_offenses(<<-RUBY)
+      it { is_expected.to match_array(array1 + array2) }
+    RUBY
+  end
+
+  it 'does not flag `contain_exactly` with mixed arguments' do
+    expect_no_offenses(<<-RUBY)
+      it { is_expected.to contain_exactly(content, *array) }
+    RUBY
+  end
+
+  include_examples 'autocorrect',
+                   'it { is_expected.to contain_exactly(*array1, *array2) }',
```

Comment:
@faucct what do you think?

Reply:
I think we can add this as an unsafe cop (can yield false positive by design) and mark it as unsafe as soon as Rubocop itself support this meta data (see rubocop-hq/rubocop#5980)

Reply:
Is skipping autocorrection when any of the arguments is a splat an option?

Reply:
Answer to self: it's not because the whole focus of the cop is when all arguments are splats.

But realistically, why would anyone use `contain_exactly(*hash1, *hash2)`? 🤔